versions.yaml

```diff
@@ -61,8 +61,7 @@ assets:
     qemu:
       description: "VMM that uses KVM"
       url: "https://github.com/kata-containers/qemu"
-      version: "741f430a960b5b67745670e8270db91aeb083c5f-29"
-      release: "19360"
+      version: "stable-2.11"
```

Comment:
Why are we still relying on `url: "https://github.com/kata-containers/qemu"` ? I thought we could point to the official qemu repo since we want a vanilla version here.

Reply:
Just because in `kata-containers/qemu` we have the `configure.sh` script which provides the flags that we use.
https://github.com/kata-containers/qemu/blob/stable-2.11/configure.sh

Reply:
Ok

Reply:
As mentioned https://github.com/kata-containers/tests/pull/182#discussion_r177658454 though, we shouldn't be storing the config script in the qemu branch itself.

So I think we can switch the URL here.